pkgs/top-level/all-packages.nix

```diff
@@ -3935,6 +3935,8 @@ with pkgs;
 
   hakrawler = callPackage ../tools/security/hakrawler { };
 
+  handheld-daemon = callPackage ../by-name/ha/handheld-daemon/package.nix { };
```

Comment:
`hhd` or `handheld-daemon`? I'm sure hhd is available.

Reply:
I wasn't sure which to choose, so I went with the more human-readable.

`services.handheldDaemon.enable` gives you a better idea of what you're doing than `services.hhd.enable`.